modules/gapi/src/backends/oak/goak_memory_adapters.cpp

```diff
@@ -27,6 +27,32 @@ MediaFrame::View OAKMediaAdapter::OAKMediaAdapter::access(MediaFrame::Access) {
 
 cv::GFrameDesc OAKMediaAdapter::OAKMediaAdapter::meta() const { return {m_fmt, m_sz}; }
 
+
+OAKRMatAdapter::OAKRMatAdapter(const cv::Size& size,
+                               int precision,
+                               std::vector<float>&& buffer) {
+    m_size = size;
+    m_precision = precision;
+    GAPI_Assert(m_precision == CV_16F);
+    m_buffer = buffer;
+
+    std::vector<int> wrapped_dims{1, 1, m_size.width, m_size.height};
```

Comment:
Is it always {1, 1 ..}?

Reply:
Currently it is. Also, I believe, there asserts on that